app/scienceapi/events/models.py

```diff
@@ -70,15 +72,26 @@ class Event(models.Model):
         help_text='Slug appended to the Mozilla Science event\'s '
                   'url that represents this event',
     )
+    is_virtual = models.BooleanField(
+        default=False,
+        help_text='When selected, events will appear on site converted'
+                  'to a visitor\'s local time.',
+        verbose_name='Virtual Event'
+    )
+
+    ALL_TIMEZONES = [(i, i) for i in pytz.all_timezones]
+    timezone = models.CharField(
+        max_length=40,
+        choices=ALL_TIMEZONES,
```

Comment:
I would say that you should probably put your choices in the form field instead. i.e. render this as a `ChoiceField` and use this as choices. There doesn't seem to be a need to do model level validation here.